src/components/NewClusterModal.js

```diff
@@ -507,7 +508,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
           style: { backgroundColor: colors.dark(0.2), borderRadius: 100, width: 'fit-content', padding: '0.75rem 1.25rem', ...styles.row }
         }, [
           span({ style: { ...styles.label, marginRight: '0.25rem' } }, ['COST:']),
```

Comment:
Non-blocking question for the future: Why do we use 'COST' (all caps) here rather than 'Cost' with `textTransform: 'uppercase'`

Reply:
Great question, I can't think of a reason and I don't see 'COST' anywhere else in the code so it's probably some tech debt? I will change it